src/redux/actions/index.jsx

```diff
@@ -0,0 +1,178 @@
+import * as ACTIONS from './actionTypes';
+
+export const fetchMovies = () => (dispatch) => {
+	dispatch(fetchMoviesStart());
+
+	fetch('http://localhost:4000/movies')
```

Comment:
It would be better to move the endpoint to a constant for the current project (usually we use env variables for that)